plugins/modules/hetzner_dns_record.py

```diff
@@ -53,8 +53,8 @@
     hetzner_token: access_token
 
 - name: Remove a new.foo.com A record
-  community.dns.hosttech_dns_record_set:
-    state: present
+  community.dns.hosttech_dns_record:
```

Comment:
```suggestion
  community.dns.hetzner_dns_record:
```
This is an example for hetzner_dns_record (see the module's filename).

Reply:
You're right, that was a bit quick of me to adjust.